File-Formats/DocIO/Word-document/Compare Word documents.md

```diff
@@ -0,0 +1,229 @@
+*	Insertions
+*	Deletions
+*	Formatting
+
+N> 1. DocIO performs word level comparison while comparing two Word documents.
+N> 2. Comparing two Word documents supported in DOCX format only.
+
+## Compare two Word documents 
+
+Compare the existing Word documents or document that is created from scratch using Compare method in .NET Word library (DocIO).
+
+The following code example illustrates how to compare two Word documents.
+
+{% tabs %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+ //Loads the original document.
+using (FileStream originalDocumentStreamPath = new FileStream("Data/OriginalDocument.docx", FileMode.Open, FileAccess.Read))
+{
+    using (WordDocument originalDocument = new WordDocument(originalDocumentStreamPath, FormatType.Docx))
+    {
+        //Loads the revised document.
+        using (FileStream revisedDocumentStreamPath = new FileStream("Data/RevisedDocument.docx", FileMode.Open, FileAccess.Read))
+        {
+            using (WordDocument revisedDocument = new WordDocument(revisedDocumentStreamPath, FormatType.Docx))
+            {
+                // Compare the original and revised Word documents.
+                originalDocument.Compare(revisedDocument);
+
+                //Saves the Word document to MemoryStream
+                MemoryStream stream = new MemoryStream();
+                originalDocument.Save(stream, FormatType.Docx);
+            }
+        }                 
+    }                           
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Loads the original document.
+using (WordDocument originalDocument = new WordDocument("Data/OriginalDocument.docx", FormatType.Docx))
+{
+    //Loads the revised document.
+    using (WordDocument revisedDocument = new WordDocument("Data/RevisedDocument.docx", FormatType.Docx))
+   {
+        /Compare original document and revised documents.
+        originalDocument.Compare(revisedDocument);
+        //Save the Word document.
+        originalDocument.Save("Result.docx");          
+    }
+}
+
+
+{% endhighlight %}
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+' Loads the original document.
+Using originalDocument As New WordDocument("Data/OriginalDocument.docx", FormatType.Docx)
+    ' Loads the revised document.
+    Using revisedDocument As New WordDocument("Data/RevisedDocument.docx", FormatType.Docx)
+        ' Compare original document and revised documents.
+        originalDocument.Compare(revisedDocument)
+        ' Save the Word document.
+        originalDocument.Save("Result.docx")
+    End Using
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+![Compare Word documents in C#](WorkingwithWordDocument_images/Compare-Word-documents-without-author.png)
+
+## Set Author and Date 
+
+Compare two Word documents by setting author and date for revisions to identify the changes clearly.
```

Comment:
Compare the two Word documents by setting the author and date for revisions to identify the changes.